### DISCUSSION TITLE
How should I customize the style of the 404 page

### DISCUSSION THREAD

**Discussion by bd-dxg** [vuejs/vitepress#1928] [Q&A] [answered]
How should I customize the style of the 404 page? 

I didn't find the relevant documents.can you help me?

**Comment by huccct** [comment, marked as answer]
I think you can do this to achieve the effect

1.✔️ Step1: You should create a new folder under **./vitepress/theme** and create a **Layout.vue** （In the Layout.vue you can define the error page you want）
![image](https://user-images.githubusercontent.com/87902123/218628507-0e764ef8-2d23-462d-8e04-19b07e33b342.png)

2.✔️ Step2: You should import  **Layout.vue** in **theme/index.ts** (In the documents, [**Like this**](https://vitepress.vuejs.org/guide/theme-introduction), So import the Layout.vue)
![image](https://user-images.githubusercontent.com/87902123/218629093-70606c90-3a58-4cd0-9873-e9c6e2843698.png)

3.✔️ Step3: Run the start command,  and you can see
![404](https://user-images.githubusercontent.com/87902123/218630691-31bc8e40-41a7-40d3-8cd8-3ffedb0acd8d.gif)

I hope my answer can help you~ 🔥

**Reply by bd-dxg** [reply]
Thank you very much for your reply. I will try your plan after work. I hope everything goes well!

**Reply by huccct** [reply]
all,right, if you succeed, hopefully mark me as the answer
